sdks/builds/wasm.mk

```diff
@@ -6,12 +6,19 @@ EMSCRIPTEN_SDK_DIR=$(TOP)/sdks/builds/toolchains/emsdk
 
 $(TOP)/sdks/builds/toolchains/emsdk:
 	git clone https://github.com/juj/emsdk.git $(EMSCRIPTEN_SDK_DIR)
+
+.stamp-wasm-checkout-and-update-emsdk: | $(EMSCRIPTEN_SDK_DIR)
```

Comment:
don't you need to somehow include the version in the stamp file? otherwise on a bump you'd not get the new version